tutorial/add-plugin/add-plugin.md

```diff
@@ -21,14 +21,18 @@ To quote them:
 ## Writing a Plugin
 
 For this, first, we need to copy the Piku from the `piku` user's home directory to root.
-TODO: explain why this is required
+We need to do this because only this instance of piku is able to use the plugins that we define.
 
 ```bash
 cp /home/piku/piku.py /root/piku.py
 ```{{execute}}
 
 We can define our plugin in the `/root/.piku/plugins/automation/__init__.py` file.
-TODO: explain what it is, why we want it/why it NEEDS to be a plugin
+
+In order to create plugins we use [https://click.palletsprojects.com/en/8.1.x/](click) which is a Python package that allows us to create command-line interfaces.
+We can define groups of commands, and then define the commands themselves with or without arguments.
+
+In this tutorial, we will create a plugin that allows us to change the timer of a Clojure app to a specific value given as an argument.
```

Comment:
I'd change "a Clojure app" to "our Clojure app" or smth -- we want to do this because this is related to our sample app, right?